src/commands/config/index.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  ChannelType,
+  ChatInputCommandInteraction,
+  GuildTextBasedChannel
+} from 'discord.js';
+import { PermissionFlagsBits } from 'discord.js';
+
+import { Command } from '../../types/command.js';
+import type {
+  CommandBuilder,
+  CommandBuilderOutput
+} from '../../types/command-builder.js';
+import { enableRandomVoiceChannelNamesConfig } from './enable-random-voice-channel-names-config.js';
+import { enableKarmaTrackingConfig } from './enable-karma-tracking-config.js';
+
+enum ConfigSubcommand {
+  EnableKarmaTracking = 'enable-karma-tracking',
+  EnableRandomVoiceChannels = 'enable-random-voice-channels'
```

Comment:
Not sure if `enable` is the correct verb, since the command does both enabling and disabling. Maybe it makes more sense to read the command as `/config random-voice-channels true` where the "config object" is  `random-voice-channels` and the boolean is the enable/disable 

Potentially this could even look like `/config random-voice-channels enable true` which would let us have other sub-subcommands and keep the whole thing consistent.

Reply:
Agreed on `enable` feeling wrong. Will rename these now and we can figure out sub-subcommands later